cmd/truss/main.go

```diff
@@ -240,33 +243,41 @@ func generateCode(cfg *truss.Config, dt deftree.Deftree, sd *svcdef.Svcdef) ([]t
 
 	genDocFiles := gendoc.GenerateDocs(dt)
 
-	genFiles := append(genGokitFiles, genDocFiles...)
+	return combineFiles(genGokitFiles, genDocFiles), nil
+}
+
+// combineFiles takes any numbers of fileSystems and combines them into one.
+func combineFiles(group ...fileSystem) fileSystem {
```

Comment:
It is possible.

Before truss was just writing from a slice, so items later in the slice would overwrite earlier items if they had the same path. Same issue, just clearer in this case.

I believe even if there was a conflict, truss should write what it can.

So I am not sure how to handle this. Options:
- Add check and just log.Warn within the function if this happens
- Return the output and an error if there was a conflict. Like `io.Reader` returns the number of bytes read and an errror. Then log after the function is called if there was an error